Podcast/SearchPeopleTableViewCell.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  SearchPeopleTableViewCell.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+protocol SearchPeopleTableViewDelegate {
+    func searchPeopleTableViewCell(cell: SearchPeopleTableViewCell, didPressFollowButtonFor user: User, newValue: Bool)
```

Comment:
whatever is implementing this delegate protocol should already have access to the user model